liblouis/compileTranslationTable.c

```diff
@@ -1391,22 +1331,23 @@ findRuleName(const CharsString *name, RuleName *ruleNames) {
 
 static int
 addRuleName(FileInfo *nested, CharsString *name, TranslationTableOffset *newRuleOffset,
-		RuleName **ruleNames, TranslationTableHeader *table) {
+		RuleName **ruleNames) {
 	int k;
 	struct RuleName *nameRule;
 	if (!(nameRule = malloc(sizeof(*nameRule) + CHARSIZE * (name->length - 1)))) {
 		compileError(nested, "not enough memory");
 		_lou_outOfMemory();
 	}
 	memset(nameRule, 0, sizeof(*nameRule));
+	// a name is a sequence of characters in the ranges 'a'..'z' and 'A'..'Z'
```

Comment:
This seems arbitrary. Was this like that before?

Reply:
No it was not like this before. But it's also not totally arbitrary. The problem is that we can't replicate the old behavior because it sucked. Firstly because we depended on other rules in the table only to determine which are valid letters to use in a name. And secondly because it allowed you yo define e.g. `.`, `-`, `1`, `2`,... as letters and then use it in a name, while these characters are also used to define repitition in multipass rules. I figured for clarity and simplicity it would be best if the new solution allows only normal letters, even if that is maybe somewhat limiting compared to how it was before.

Reply:
OK, fair enough. Maybe this should be added in the doc somewhere. I'll open a separate issue for this